tsbrowse/model.py

```diff
@@ -54,20 +64,34 @@ def __init__(self, tsbrowse_path):
             array_names -= set(ragged_array_names)
             array_names -= {"metadata_schema"}
             array_names -= {f"{name}_offset" for name in ragged_array_names}
-            arrays = {name: root[table_name][name][:] for name in array_names}
-            ragged_array_names -= {"metadata"}
+            arrays = {}
```

Comment:
This feels like we're reproducing tszip code to me - why do we need to do this?

Reply:
I guess your suggesting that for a fixed list of `table/col` we use tskit, and the rest load using zarr? Yes, would probably be cleaner.